src/entrypoints/my-change-url.ts

```diff
@@ -26,6 +19,9 @@ class MyChangeUrl extends LitElement {
   @query("my-url-input") private _urlInput?: MyUrlInputMain;
 
   createRenderRoot() {
+    while (this.lastChild) {
```

Comment:
This is to remove the placeholder elements. New Lit appends by default.

@bramkragten is this the best practice to do this?

Reply:
I think so...